app/policies/application_policy.rb

```diff
@@ -1,3 +1,4 @@
+# Application policy class
 class ApplicationPolicy < ActionPolicy::Base
   # make :manage? a real catch-all
```

Comment:
I'm curious about this comment. Is it saying that by default `index?` and `create?` delegate to `manage?`, and if so, can we achieve this instead by using an alias?

Or is it a TODO saying we should set a global `default_rule :manage?` here? (Which is okay with me too.)

Reply:
It's because index? and create? are defined in the superclass ActionPolicy::Base so setting a default rule won't actually work here (https://actionpolicy.evilmartians.io/#/custom_policy?id=custom-base-policy). 
We can use aliases but that loses some flexibility if we want to show different error messages in the future. But since we're not differentiating right now, I'm happy to move them over to aliases.

Reply:
Thanks for the link!